fog/sample-paykit/src/cached_tx_data/mod.rs

```diff
@@ -130,7 +130,7 @@ impl CachedTxData {
             owned_tx_outs: Default::default(),
             key_image_data_completeness: BlockCount::MAX,
             latest_global_txo_count: 0,
-            latest_block_version: 1,
+            latest_block_version: 0,
```

Comment:
Can it use the `BlockVersion` constant so that it becomes greppable?

Reply:
This was answered [here](https://github.com/mobilecoinfoundation/mobilecoin/pull/1734#pullrequestreview-926674436)